iroh-net/src/magicsock/node_map.rs

```diff
@@ -389,6 +405,25 @@ impl NodeMapInner {
             .map(|ep| ep.info(Instant::now()))
     }
 
+    /// Returns a stream of [`ConnectionType`].
+    ///
+    /// Sends the current [`ConnectionType`] whenever any changes to the
+    /// connection type for `public_key` has occured.
+    ///
+    /// Errors:
```

Comment:
FWIW rust doc conventions/rules/style-guides/whatever use `# Errors` for this.

Reply:
oops thank you, that was what i was going for